src/main/java/net/cytonic/cytosis/events/ServerEventListeners.java

```diff
@@ -32,7 +32,7 @@ public static void initServerEvents() {
         Logger.info("Registering player chat event.");
         Cytosis.getEventHandler().registerGlobalEvent(PlayerChatEvent.class, event -> {
             final Player player = event.getPlayer();
-            Cytosis.getDatabase().addChat(player.getUuid(),event.getMessage());
+            Cytosis.getManager().getDatabase().addChat(player.getUuid(),event.getMessage());
         });
```

Comment:
I can't help but notice that you dont check if it is enabled in the config. `CytosisSettings.LOG_PLAYER_CHAT`